.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @seek-oss/indirect
```

Reminder to change this

---

Open to changing this in the PR if we can come to an agreement

---

What about:

```suggestion
* @seek-oss/sig-backend-tooling @seek-oss/sku-maintainers 
```

(and adjusting the admin access as per ^)